### PR TITLE
[FastPR][All] Upgrade minimum Python version to 3.8

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
Dear @KratosMultiphysics/all , as described in #10420 we require to upgrade minimum Python version to 3.8. Note that the current version we are using, the 3.6, reached it's end of life past December.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Additionaly, Visual Studio is required to compile in *Windows*.
         ```
 
     - Python 
-        You will need at least *Python* 3.7 (recommended 3.8/3.9/3.10) in your computer in order to compile *Kratos*. You can download python from its official webpage:
+        You will need at least *Python* 3.8 (recommended 3.8/3.9/3.10) in your computer in order to compile *Kratos*. You can download python from its official webpage:
```
Comment (philbucher) [inline INSTALL.md:147]: you can add 3.11, it was released recently

Comment (rubenzorrilla) [reply INSTALL.md:147]: I'll wait until we try it (just in case). We can add it as soon as we use it in one of our CI(s). Makes sense for you?

Comment (roigcarlo) [reply INSTALL.md:147]: Wait a couple of weeks until I add it to the rel dockers pls. I will do it in the same PR

**Review by roigcarlo — APPROVED** [review]

**Comment by pooyan-dadvand** [comment]
Just to note that my comment in #10420 about not moving to 3.8 was completely ignored...

And as a side effect our CI is failing to compile the current master until we reconfigure the servers....

**Pull request merged**